### ISSUE TITLE
[Backport v2.6] Rancher container is restarting every 15 seconds

### ISSUE THREAD

**Issue by rancherbot** [rancher/rancher#38918]
This is a backport issue for https://github.com/rancher/rancher/issues/36238, automatically created via rancherbot by @kinarashah

Original issue description:

**Rancher Server Setup**
- Rancher version: v2.6.3
- Installation option (Docker install/Helm Chart): Docker install

**Describe the bug**

After I restarted my ubuntu vm, my Rancher UI docker container is restarting every 15 seconds

Here is the log:
```
2022/01/22 11:40:19 [INFO] Rancher version v2.6.3 (3c1d5fac3) is starting
2022/01/22 11:40:19 [INFO] Rancher arguments {ACMEDomains:[] AddLocal:true Embedded:false BindHost: HTTPListenPort:80 HTTPSListenPort:443 K8sMode:auto Debug:false Trace:false NoCACerts:false AuditLogPath:/var/log/auditlog/rancher-api-audit.log AuditLogMaxage:10 AuditLogMaxsize:100 AuditLogMaxbackup:10 AuditLevel:0 Features: ClusterRegistry:}
2022/01/22 11:40:19 [INFO] Listening on /tmp/log.sock
2022/01/22 11:40:19 [INFO] Waiting for server to become available: Get "https://127.0.0.1:6443/version?timeout=15m0s": dial tcp 127.0.0.1:6443: connect: connection refused
2022/01/22 11:40:21 [INFO] Waiting for server to become available: Get "https://127.0.0.1:6443/version?timeout=15m0s": dial tcp 127.0.0.1:6443: connect: connection refused
2022/01/22 11:40:23 [INFO] Waiting for server to become available: the server is currently unable to handle the request
2022/01/22 11:40:25 [INFO] Waiting for server to become available: an error on the server ("apiserver not ready") has prevented the request from succeeding
2022/01/22 11:40:27 [INFO] Waiting for server to become available: an error on the server ("apiserver not ready") has prevented the request from succeeding
2022/01/22 11:40:42 [INFO] Running in single server mode, will not peer connections
2022/01/22 11:40:43 [INFO] Applying CRD features.management.cattle.io
2022/01/22 11:41:29 [ERROR] unable to update feature harvester in initialize features: Put "https://127.0.0.1:6443/apis/management.cattle.io/v3/features/harvester": EOF
2022/01/22 11:41:29 [FATAL] k3s exited with: exit status 255
```
Any advice how to make it work again?

**Comment by kinarashah** [comment]
Available to test with v2.6-head once https://drone-publish.rancher.io/rancher/rancher/8316 is green.

**Comment by thaneunsoo** [comment]
Closing this issue based on results from [39072](https://github.com/rancher/rancher/issues/39072)
